### PR TITLE
AT_04.23.10 | Verify Notes/Remark input field has a “Booking notes” text placeholder.

### PR DESCRIPTION
**Pull request by Forostinko**
https://trello.com/c/JftbuQGZ/755-at042310-verify-notes-remark-label-is-present-and-visible

### CODE REVIEW

File: cypress/pageObjects/CreateBookingPage.js [hunk 1]
```diff
@@ -64,7 +64,8 @@ class CreateBookingPage {
     getMainPassengerSelectedSeatByDefault = () => cy.get('div[class="col-lg-12 passenger-row"] span[class="seat-number"]')
     getPassengerDetailsAssignedSeats = () => cy.get('span.seat-number')
     getSelectedDialCode = () => cy.get('.iti__selected-dial-code');
-    getDialCodeArrow = () => cy.get('.iti__selected-flag')
+    getDialCodeArrow = () => cy.get('.iti__selected-flag');
+    getPlaceholderBookingNotes = () => cy.get('textarea#booking_notes')
```
Comment (Rom-T) [inline cypress/pageObjects/CreateBookingPage.js:68]: Please change the name of the element to getNotesInputField

**Review by Rom-T — APPROVED** [review]

**Review by SvetlanaKanel — CHANGES_REQUESTED** [review]
You need to verify the label but you are verifying the placeholder

**Review by Rom-T — CHANGES_REQUESTED** [review]
Yes, I missed. The test is on the inputField, but on the label.

**Review by Rom-T — APPROVED** [review]

**Comment by Forostinko** [comment]
>" You need to verify the label but you are verifying the placeholder @SvetlanaKanel  fixed

**Comment by Forostinko** [comment]
> You need to verify the label but you are verifying the placeholder
@SvetlanaKanel fised

**Review by Rom-T — APPROVED** [review]

**Review by SvetlanaKanel — APPROVED** [review]

**Pull request closed without merging**